### PR TITLE
Mensuration Symbols

### PR DESCRIPTION
**Pull request by seemantk**
Added BravuraText font as well as a demo in the html file.
mensurations

### CODE REVIEW

**Comment by seemantk** [comment]
Note, that this branch is branched off the measure-numbers branch in pr #58

**Comment by seemantk** [comment]
@curran, I believe this is ready.  Should close issue #4

File: js/notesbook.js [hunk 1]
```diff
@@ -166,7 +206,24 @@ function NotesBook() {
...
+  } // reflinesRender()
+
+
+  function mensurationsRender(selection) {
+      selection
+          .call(mensurationsAxis)
+        .selectAll(".tick")
+          .each(function(d, i) {
+              var self = d3.select(this)
+                , sym = data.barlines[i].mensuration
```
Comment (curran) [inline js/notesbook.js:218]: Nice.

**Comment by curran** [comment]
Looks great! I don't see any mensuration changes in the default work, is there a specific other work you've been using for testing?

**Pull request merged**